src/UI/class-settings-page.php

```diff
@@ -941,22 +982,24 @@ public function validate_options( array $input ): array {
 
 		$input['api_secret'] = sanitize_text_field( $input['api_secret'] );
 
-		if ( ! empty( $input['metadata_secret'] ) ) {
+		if ( '' !== $input['metadata_secret'] ) {
 			if ( strlen( $input['metadata_secret'] ) !== 10 ) {
 				add_settings_error(
 					Parsely::OPTIONS_KEY,
 					'metadata_secret',
 					__( 'Metadata secret is incorrect. Please contact Parse.ly support!', 'wp-parsely' )
 				);
-			} elseif ( isset( $input['parsely_wipe_metadata_cache'] ) && 'true' === $input['parsely_wipe_metadata_cache'] ) {
+			} elseif (
+				isset( $input['parsely_wipe_metadata_cache'] ) && 'true' === $input['parsely_wipe_metadata_cache'] // @phpstan-ignore-line
```

Comment:
`parsely_wipe_metadata_cache` is defined as bool in our array structure but here we are using true as string so added ignore comment to avoid this error (just a quicker and faster solution)